src/components/entity.jsx

```diff
@@ -0,0 +1,126 @@
+import DocumentTitle from 'react-document-title'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import config from '../config'
+import { entityShape } from '../openfisca-proptypes'
+
+
+class Entity extends React.Component {
+  static propTypes = {
+    countryPackageName: PropTypes.string.isRequired,
+    countryPackageVersion: PropTypes.string.isRequired,
+    intl: intlShape.isRequired,
+    entityId: PropTypes.string.isRequired,
+    entity: entityShape.isRequired,
+  };
+
+  
+  render() {
+    const { entityId, entity } = this.props
+
+    return (
+      <DocumentTitle title={entityId + ' — ' + this.props.intl.formatMessage({ id: 'appName' })}>
+        <section>
+          <h1><code>{ entityId }</code></h1>
+          {
+            entity.plural
+            ? <span>
+                <h5><code>{ entity.plural }</code></h5>
```

Comment:
`Plural: { entity.plural }` ?

This is not super clear right now:
![image](https://user-images.githubusercontent.com/11834997/46259522-cca3c680-c4da-11e8-834f-ca4b66cc1892.png)